src/sections/qualitat.html

```diff
@@ -0,0 +1,43 @@
+<div class="h-full py-10 px-8 md:py-20 md:px-10 xl:py-52 xl:px-32 bg-darkBlue 2xl: 2xl:grid justify-center">
+    <div> 
+        <p class="text-white">
+            <span class="font-railwayBold text-xl leading-3">Qualität steht bei uns an erster Stelle. Mit unserem <br class="hidden lg:block">
+                <span class="text-green">Trainingsprogramm</span> direkt Vorort werden die <br class="hidden lg:block"> juniorigen Mitarbeitenden auf das projektgeschäft <br> vorbereitet.</span> 
+            <br> <br>
+            <span class="font-extralight text-sm leading-none">
+                kurze Einführung und den Bezug zu MW darstellen At vero eos et accusam et justo duo <br class="hidden sm:block">
+                dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est L vero eos et <br class="hidden sm:block">
+                accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata <br class="hidden sm:block">
+                sanctus est L
+            </span>
+        </p>
+    </div>
+    
+    <div class="grid lg:flex justify-center w-full gap-16 py-10 text-white">
+        <div>
+            <section class="flex flex-col w-full">
+            <div class="flex justify-between px-3 gap-28">
+                <span class=" font-railwayBold">coding Skills</span>
+                <span class="text-xl font-railwayBold">+</span>
+            </div>
+            <span class="h-1 w-full bg-green"></span>
+            </section>
+        </div>
+        
+        <div>
+            <section class="flex flex-col w-full">
+            <div class="flex justify-between px-3 gap-28">
+                <span class=" font-railwayBold">Kommunikations- und Teamfähigkeit</span>
+                <span class="text-xl font-railwayBold">+</span>
+            </div>
+            <span class="h-1 w-full bg-green"></span>
```

Comment:
May be you can try to make those green lines equal as it is in the figma design.